BlueprintUI/Sources/Layout/ConstrainedSize.swift

```diff
@@ -1,21 +1,44 @@
 import UIKit
 
-/// Constrains the measured size of the content element.
+/// Constrains the measured size of the contained element in the ranges specified by the `width` and `height` properties.
+///
+/// There are several constraint types available for each axis. See `ConstrainedSize.Constraint` for a full list and in-depth
+/// descriptions of each.
+///
+/// Notes
```

Comment:
I documented this existing behavior, but I don't love it... thoughts on changing the behavior so the max returned is the max of the measurement constrained OR the constrained size, whichever is smaller?

Reply:
(You can see how confusing this existing behaviour gets in the tests, in the `test_in_atMost` method.)